lib/main.dart

```diff
@@ -32,7 +34,15 @@ class _TenTenOneState extends State<TenTenOneApp> {
   @override
   void initState() {
     super.initState();
-    _callInitWallet();
+
+    _callInitWallet()
+        .then((value) => FLog.info(text: "Successfully initialised wallet"))
+        .catchError((error) => {
+              if (error is FfiException)
+                {FLog.error(text: "Wallet failed to initialise: Error: " + error.message)}
+              else
+                {FLog.error(text: "Wallet failed to initialise: Unknown error")}
```

Comment:
is there a way of having a generic handler that handles uncaught exceptions/errors?

Reply:
Yes, https://github.com/itchysats/10101/pull/104/commits/192a5d532f6a2bde4f6c71795b25cdde7f8b543f

I've added a rudimentary implementation, but we should spent a little bit more time on how to handle errors e.g.

- automatically send the logs to us https://github.com/itchysats/10101/issues/105
- design an error screen
- etc.